### DISCUSSION TITLE
New Event with a Defined Task via Front-End Submit or Import

### DISCUSSION THREAD

**Discussion by roysinn** [liedekef/events-made-easy#95] [Ideas]
Our organization provides free health and wellness workshops to seniors. Groups can request a workshop (from a defined list), date, time, etc. Our volunteer facilitators can then sign up to present the workshop. We have a clumsy system for doing this now but EME seems to have all the tools to do this more effectively. Here's my question(s): 

Is it possible to use the front-end submit plug-in to create events with a "Facilitator" task that matches the event date and time? If not, would it be difficult to modify the plug-in to do so? I do have PHP programming experience but my skills are dated. Just wondering if I would be looking at a major or minor re-write or just some modifications. 

My second, but much less desirable solution, is to use the CSV import for a new event. But can it be used to create a task for that new event as well? 

Many thanks in advance.

Cheers,
Roy

**Comment by liedekef** [comment]
Currently tasks are not imported. Nor is it possible to use the frontend form for this, I'd consider that a minor rewrite.

**Reply by roysinn** [reply]
Thanks Franky. A "minor rewrite" sounds do-able.

**Reply by liedekef** [reply]
Well, feel free to take a stab at it :-)
